### ISSUE TITLE
NAO SKD for nvidia jetson 

### ISSUE THREAD

**Issue by ngeony33** [OlalekanIsola/alive#1]
Hello! Did you have any succes with Nao python SDK for Nvidia Jetson?

**Comment by OlalekanIsola** [comment]
No, i didn't. I am still trying to figure out what to do.Any suggestions?
i tried  Libqi, it supports other versions of  Ubuntu except Melodic

**Comment by ngeony33** [comment]
I'm trying to install the old dependencies on ubuntu 18 but i didnt have any succes...
